File: app/src/main/java/com/ihadzhi/eatlimination/ui/ContainerInteractor.java
package com.ihadzhi.eatlimination.ui;

import androidx.annotation.StringRes;

public interface ContainerInteractor {

    void hideBottomNavigation();

    void showBottomNavigation();

    void showLoadingIndicator();

    void hideLoadingIndicator();

    void hideSoftKeyboard();

    void errorGeneric();

    void error(@StringRes int errorMessage);

}

File: app/src/main/java/com/ihadzhi/eatlimination/ui/ContainerInteractor.kt
package com.ihadzhi.eatlimination.ui

import androidx.annotation.StringRes

interface ContainerInteractor {
    fun hideBottomNavigation()
    fun showBottomNavigation()
    fun showLoadingIndicator()
    fun hideLoadingIndicator()
    fun hideSoftKeyboard()
    fun errorGeneric()
    fun error(@StringRes errorMessage: Int)
}